src/gpio.jl

```diff
@@ -0,0 +1,28 @@
+typealias gpio_context Ptr{Void}
+
+@enum(gpio_dir_t,
+  GPIO_OUT = 0,
+  GPIO_IN = 1,
+  GPIO_OUT_HIGH = 2,
+  GPIO_OUT_LOW = 3
+)
+
+function gpio_init(pin::Int)
```

Comment:
I believe this should be `Int32` in case this gets run on a 64 bit arch. `Cint` should still be 32 bit on a 64 bit arch but correct me if I am wrong.